### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by shuiblue**

### CODE REVIEW

**Comment by shuiblue** [comment]
https://www.digitalocean.com/community/tutorials/how-to-serve-flask-applications-with-uwsgi-and-nginx-on-ubuntu-16-04

File: README.md [hunk 1]
```diff
@@ -4,56 +4,63 @@ Website: http://forks-insight.com or http://128.2.112.151
...
 
 Language: Python3
 
 Framework: Flask
 
 Database: mongo
 
+Http server: uwsgi  (./config.ini - Config for using uWSGI)
 
+Reverse Agent: nginx (/etc/init.d/nginx)
```
Comment (FancyCoder0) [inline README.md]: delete /etc/init.d/nginx

File: README.md [hunk 2]
```diff
@@ -4,56 +4,63 @@ Website: http://forks-insight.com or http://128.2.112.151
...
-3. Run on localhost: python manage.py runserver
+INFOX 
 
 Language: Python3
 
 Framework: Flask
 
 Database: mongo
 
+Http server: uwsgi  (./config.ini - Config for using uWSGI)
```
Comment (FancyCoder0) [inline README.md]: delete ./config.ini - Config for using uWSGI (already in following description)

File: README.md [hunk 3]
```diff
@@ -4,56 +4,63 @@ Website: http://forks-insight.com or http://128.2.112.151
...
 
+ - install conda (package manager)
+    https://conda.io/docs/user-guide/install/download.html
+    Download Anaconda
 
+ - install dependencies (https://github.com/FancyCoder0/INFOX/blob/master/environment.yaml)
+   conda env create -f environment.yaml
 
-./config.ini - Config for using uWSGI
+ - install mongo (install brew mongo)
```
Comment (FancyCoder0) [inline README.md]: change to : install mongodb

File: README.md [hunk 4]
```diff
@@ -4,56 +4,63 @@ Website: http://forks-insight.com or http://128.2.112.151
...
 
+ - install conda (package manager)
+    https://conda.io/docs/user-guide/install/download.html
+    Download Anaconda
 
+ - install dependencies (https://github.com/FancyCoder0/INFOX/blob/master/environment.yaml)
+   conda env create -f environment.yaml
 
-./config.ini - Config for using uWSGI
+ - install mongodb (install brew mongo)
```
Comment (FancyCoder0) [inline README.md]: install brew mongo is wrong
correct is: brew install mongodb 
My idea is that just delete it.

**Pull request merged**